tests/testthat/test-ProjectExplicit.R

```diff
@@ -0,0 +1,13 @@
+library(ProjectExplicit)
+
+context("Merging files functionality")
+
+test_that("merging files works", {
+  merge_data_files("C:/Users/mrkon/Desktop/UvA/CoPY/CoPY_2.0/analysis/data/data_stroop")
+  
+  #checks that the original number of data files is the same as  length of the merged data frame
+  expect_equal(length(data_stroop), nrow(stroop_metrics))
+  
+  #checks that the participant ID and corresponding data are correctly merged with the original data
+  expect_equal(data_stroop[[1]]$uniqueSubID[1], stroop_metrics$uniqueSubID[1])
+})
```

Comment:
Since I am working in python and not R I do not feel very comfortable commenting on your code. What I can see is that you used the testthat function from R to check whether your files are correctly merged. 

I am also not sure whether we can already download each others packages to actually run the test functions?
Well I hope yours works, mine doesn't 👎 

Sorry for the crappy review.